portal_pages/tests/test_user_forms.py

```diff
@@ -96,6 +97,29 @@ def test_marketplace_entry_form_invalid_honeypot(self):
         self.assertEqual(form.errors['__all__'], ['Invalid submission'])
         self.assertEqual(MarketplaceEntryPage.objects.all().count(), 0)
 
+    def test_marketplace_entry_good_image(self):
+        test_file = open('portal_pages/tests/files/goodimage.jpg', 'rb')
+        good_image = SimpleUploadedFile(test_file.name, test_file.read())
+        self.marketplace_entry_form_data['file'] = good_image
+        resp = self.client.post(self.marketplace_submission_url, self.marketplace_entry_form_data)
+        self.assertEqual(resp.status_code, 302)
+        expected_redirect = self.marketplace_index_page.url + self.marketplace_index_page.reverse_subpage('thanks')
+        self.assertRedirects(resp, expected_redirect)
+
+    def test_marketplace_entry_bad_image_type(self):
+        test_file = open('portal_pages/tests/files/badimage.bmp', 'rb') # File types accepted are GIF/JPG/PNG, this should error
+        good_image = SimpleUploadedFile(test_file.name, test_file.read())
+        self.marketplace_entry_form_data['file'] = good_image
+        resp = self.client.post(self.marketplace_submission_url, self.marketplace_entry_form_data)
+        self.assertEqual(resp.status_code, 200)
+
+    def test_marketplace_entry_animated_gif(self):
+        test_file = open('portal_pages/tests/files/animated.gif', 'rb') # Animated gifs are not supported, this should error
+        good_image = SimpleUploadedFile(test_file.name, test_file.read())
+        self.marketplace_entry_form_data['file'] = good_image
+        resp = self.client.post(self.marketplace_submission_url, self.marketplace_entry_form_data)
+        self.assertEqual(resp.status_code, 200)
```

Comment:
Same comment as above here to verify the reason for the "failing" 200 is due to what we expect and we're telling the user about it...